datalogistik/dataset.py

```diff
@@ -297,7 +317,9 @@ def get_raw_tpc_dataset_spec(self, table):
         # does not have the extra column at the end here)
         return dataset_read_format, pa.schema(column_types.copy())
 
-    def get_table_dataset(self, table=None):
+    def get_table_dataset(self, table: Table = None):
```

Comment:
```suggestion
    def get_table_dataset(self, table: Table = None) -> Dataset:
```
I think? Not sure; there are a lot of things called "dataset" here

Reply:
I tried this but I got an error that it couldn't find `Dataset`! This might have something to do with my old python version because I think something was added related to this